scripts/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+MAINTEX_NAME="main"
+
+show_usage() {
```

Comment:
If your sh script includes a Shebang, you should not execute it using `bash build.sh`.

Reply:
I believe while it's true that a shebang specifies which interpreter should execute the script, there's nothing wrong with explicitly using bash to run a script that includes a shebang line. Using bash script.sh will work perfectly fine - it just bypasses the shebang specification.

Reply:
That's certainly not a problem; I just prefer the UNIX style. 😂